tests/MarkdownRendererSettingsTest.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+namespace Spatie\LaravelMarkdown\Tests;
+
+use League\CommonMark\Extension\CommonMark\Node\Block\ThematicBreak;
+use League\CommonMark\MarkdownConverter;
+use League\CommonMark\Node\Node;
+use League\CommonMark\Renderer\ChildNodeRendererInterface;
+use League\CommonMark\Renderer\NodeRendererInterface;
+use League\CommonMark\Util\HtmlElement;
+use Spatie\LaravelMarkdown\MarkdownRenderer;
+use Spatie\Snapshots\MatchesSnapshots;
+
+class MarkdownRendererSettingsTest extends TestCase
+{
+    use MatchesSnapshots;
+
+    /** @test */
+    public function it_can_modify_highlight_code_option()
+    {
+        $markdownRenderer = $this->markdownRenderer();
```

Comment:
It's hard to read this test. Could you add some new lines, so there is some "breathing space"?

Make also introduce a function, and move everything you do with reflection in that function. Give that function a descriptive name of what is done.

Reply:
Good call - introduced a `getProtectedPropertyValue` method and it's simplified all the tests.